### PR TITLE
Feature/add docs

### PR DESCRIPTION
**Pull request by giovannicuriel**
* **Please check if the PR fulfills these requirements**
- [NA] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Docs update

* **What is the current behavior?** (You can also link to an open issue here)
No reference for environment variables for services

* **What is the new behavior (if this is a feature change)?**
A page listing all available environment variables was added.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No.

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
This is related to dojot/dojot#1042

* **Other information**:

### CODE REVIEW

File: docs/source/envvars.rst [hunk 1]
```diff
@@ -0,0 +1,494 @@
...
+    # Comma separated list of Kafka instances to be used for Kafka client
+    # bootstrapping.
+    export KAFKA_HOSTS="kafka:9092"
+    # Kafka group ID for consumers. It is advisible that this is the same
+    # for instances of the same service and different for each service.
+    export KAFKA_GROUP_ID="kafka"
+    # Amount of time (in seconds) that the service will wait before attempting
+    # to subscribe to a set of Kafka topics.
+    export DOJOT_KAFKA_SUBSCRIPTION_HOLDOFF="2"
+    # Time between each poll
```
Comment (rascaraficci) [inline docs/source/envvars.rst]: milliseconds?

File: docs/source/envvars.rst [hunk 2]
```diff
@@ -0,0 +1,494 @@
...
+
+    # Where API gateway is
+    export AUTH_KONG_URL="http://kong:8001"
+    export AUTH_TOKEN_EXP=420
+
+    # Auth e-mail configuration
+
+    # Flag indicating whether there should be an e-mail or not. If this var
+    # is anything but "NOEMAIL", then an email is sent to the user whenever
+    she/he needs to opeate the password.
```
Comment (rascaraficci) [inline docs/source/envvars.rst]: comment mark #?

File: docs/source/envvars.rst [hunk 3]
```diff
@@ -0,0 +1,494 @@
...
+
+    # Passowrd associated to the field right above this.
+    export AUTH_EMAIL_PASSWD=""
+
+    # ?
+    export AUTH_RESET_PWD_VIEW=""
+
+    # If not using an email, then the default password will be this one.
+    export AUTH_USER_TMP_PWD="temppwd"
+    # Period of time which the password request is deeemed valid.
```
Comment (rascaraficci) [inline docs/source/envvars.rst]: seconds?

File: docs/source/envvars.rst [hunk 4]
```diff
@@ -0,0 +1,494 @@
...
+    # Zookeeper configuration
+    export ZOOKEEPER_HOST="zookeeper"
+    export ZOOKEEPER_PORT=2181
+
+    # Port used by ZeroMQ
+    export ZEROMQ_PORT=5556
+
+    # Timeout value for locked variables. After this period, the variable
+    # will be automatically unlocked
+    export HOLD_LOCK_TIMEOUT=10000
```
Comment (rascaraficci) [inline docs/source/envvars.rst]: milliseconds?

File: docs/source/envvars.rst [hunk 5]
```diff
@@ -0,0 +1,494 @@
...
+
+    # Timeout value for wait operations for locks. After this period, the client
+    # will be automatically freed, but with no data.
+    export WAIT_LOCK_TIMEOUT=30000
```
Comment (rascaraficci) [inline docs/source/envvars.rst]: milliseconds?

File: docs/source/index.rst [hunk 6]
```diff
@@ -1,20 +1,83 @@
-.. docker-compose documentation master file, created by
-   sphinx-quickstart on Wed Apr 10 14:02:48 2019.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+DeviceManager
```
Comment (rascaraficci) [inline docs/source/index.rst]: I don't understand why this section is here.

File: docs/source/envvars.rst (hunk 1 above)
Comment (mprevide) [reply docs/source/envvars.rst]: This use time.sleep(secs) from python

File: docs/source/envvars.rst (hunk 3 above)
Comment (mprevide) [reply docs/source/envvars.rst]: it's in minutes

File: docs/source/envvars.rst (hunk 4 above)
Comment (mprevide) [reply docs/source/envvars.rst]: yes

File: docs/source/envvars.rst (hunk 5 above)
Comment (mprevide) [reply docs/source/envvars.rst]: yes

File: docs/source/index.rst (hunk 6 above)
Comment (mprevide) [reply docs/source/index.rst]: I think it was a mistake

Comment (mprevide) [reply docs/source/index.rst]: I will remove the file

**Comment by giovannicuriel** [comment]
Hey

teste

File: docs/source/envvars.rst [hunk 7]
```diff
@@ -1,15 +1,501 @@
...
 
 .. code-block:: bash
 
+    # Redis instance used by Data Broker
+    export DATABROKER_CACHE_HOST="data-broker-redis"
+
+    # Kafka configuration. These are used for topic creation.
+    export DATABROKER_KAFKA_ADDRESS="kafka"
+    export DATABROKER_KAFKA_PORT=9092
+    export DATABROKER_ZOOKEEPER_HOST="zookeeper:2181"
```
Comment (eduardogmisiuk) [inline docs/source/envvars.rst]: I didn't find anything in Data Broker code that uses the variable DATABROKER_ZOOKEEPER_HOST. I am removing it from Data Broker code and documentation, we should remove it from here too.

File: docs/source/envvars.rst [hunk 8]
```diff
@@ -1,15 +1,501 @@
...
\ No newline at end of file
+    export LOG_LEVEL="info"
+
+    # Data Broker healthcheck configuration
+    # All values are in miliseconds and indicate how much time it will
+    # wait before checking a particular state again.
+    export HC_UPTIME_TIMEOUT=300000
+    export HC_MEMORY_USAGE_TIMEOUT=300000
+    export HC_CPU_USAGE_TIMEOUT=300000
+    export HC_MONGODB_TIMEOUT=30000
```
Comment (eduardogmisiuk) [inline docs/source/envvars.rst]: We don't use MongoDB in Data Broker (anymore?), neither do we use this variable in the code. I am removing it from Data Broker code and documentation, we should remove it from here too.

File: docs/source/envvars.rst [hunk 9]
```diff
@@ -1,15 +1,501 @@
...
+
+    # Comma separated list of Kafka instances to be used for Kafka client
+    # bootstrapping.
+    export KAFKA_HOSTS="kafka:9092"
+    # Kafka group ID for consumers. It is advisible that this is the same
+    # for instances of the same service and different for each service.
+    export KAFKA_GROUP_ID="kafka"
+    # Amount of time (in seconds) that the service will wait before attempting
+    # to subscribe to a set of Kafka topics.
+    export DOJOT_KAFKA_SUBSCRIPTION_HOLDOFF="2"
```
Comment (eduardogmisiuk) [inline docs/source/envvars.rst]: This variable seems to have been changed to DOJOT_SUBSCRIPTION_HOLDOFF, in ms

Comment (eduardogmisiuk) [reply docs/source/envvars.rst]: And with default value in 2500

**Pull request merged**